### DISCUSSION TITLE
Is it possible to view and/or disable the download counter?

### DISCUSSION THREAD

**Discussion by harryhamburger** [aclap-dev/video-downloadhelper#1360] [Help and Q&A]
### Your question

I don't like the idea that VDH counts the number of downloads, as it is not made clear where that information is stored and who is able to access it in what detail, and if VDH actually phones home. It displays a congratulation message at times, stating the number of downloads, which is annoying to me. Has anyone got an idea about this issue? My version is a licensed one.

### Addon & CoApp version and details

Firefox Version 131.0.3 (64-Bit), I cannot find the VDH version with these instructions. I assume it is the lastes release, as my settings are on automatic update.

**Comment by jcfrog** [comment]
Hi,
This information is 100% local, we will not track you and we will never call you.

**Reply by harryhamburger** [reply]
But maybe you could tell me where the information is stored in the system...?

**Comment by jcfrog** [comment]
This is stored in Firefox' internal memory (session storage).

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/user-attachments/assets/40ac447e-f5f1-4362-a278-2b3f182198e7)
![#02](https://github.com/user-attachments/assets/6ecbee60-803c-4314-be20-109c42faadef)
![#03](https://github.com/user-attachments/assets/69a7ee35-f904-4e48-81d1-9a6f68489205)

**Reply by GrampaWildWilly** [reply]
You ought to read what gets copied to the clipboard before you post it.  I always censor my license number & I delete all the download history information.  Those bits are usually entirely unnecessary in diagnosing a problem.
